src/components/Markings/DrawingClassifierSubject.js

```diff
@@ -113,36 +90,24 @@ class ImageWithSvgOverlay extends Component {
     }
 
     render() {
-        const pathPrefix = Platform.OS === 'android' ? 'file://' : ''
```

Comment:
Where has this, and other items for this component, been moved to? It seems this component has changed significantly.

Reply:
So before this refactor, there were two components that essentially did the same thing: `ImageWithSvgOverlay` and `MarkableImage`. They both rendered SVGs over an image, but `MarkableImage` also allowed the user to draw SVGs over the image as well.

As a part of this refactor, I changed `ImageWithSvgOverlay` to `DrawingClassifierSubject`, who's purpose is to display the `DrawingToolView`: which contains the subject and all drawing and SVG logic. And display the blurview and already seen banner over the subject.

In addition, I modified `MarkableImage` to be drawing enabled or disabled, so it can essentially function exactly the same as `ImageWithSvgOverlay` used to function.

So, in short, the functionality wasn't really moved anywhere. It already existed in `MarkableImage` and I'm just leveraging that now